nightlies/cron_wrapper

```diff
@@ -37,6 +46,10 @@ if [ $code != 0 ] ; then
 else
         # normal exit: just log stdout
         DATE=$(date)
-        echo -n "$DATE: " >> $LOG
-        cat $STDOUT  >> $LOG
+        # if flock failed, define a new $LOG and keep runing
+        flock 200 || $LOG
```

Comment:
@dmick 
I think this line 50 is wrong, should be `flock 200 || LOG="/var/log/crontab-nightlies-log/crontab_lock_problem.log"`